components/style/stylist.rs

```diff
@@ -1065,6 +1065,21 @@ impl Stylist {
                                      CascadeFlags::empty(),
                                      self.quirks_mode))
     }
+
+    /// Accessor for a shared reference to the device.
+    pub fn device(&self) -> &Arc<Device> {
```

Comment:
Let's make this just return `&Device`. The `Arc` should ideally just be an implementation detail.